async-stream/src/lib.rs

```diff
@@ -155,7 +155,17 @@
 //! `sender.send(value)` stores the value that cell and yields back to the
 //! caller.
 //!
+//! # Limitations
+//!
+//! `async-stream` suffers from the same limitations as the [`proc-macro-hack`]
+//! crate. Primarily, nesting support must be implemented using a `TT-muncher`.
+//! If large `stream!` blocks are used, the caller will be required to add
+//! `#![recursion_limit = "..."]` to their crate.
+//!
+//! A `stream!` macro may only contain up to 64 macro invocations.
+//!
 //! [`Stream`]: https://docs.rs/futures-core/*/futures_core/stream/trait.Stream.html
+//! [`proc-macro-hack`]: https://github.com/dtolnay/proc-macro-hack/
```

Comment:
This limitation is no longer exists.

```suggestion
//! [`Stream`]: https://docs.rs/futures-core/*/futures_core/stream/trait.Stream.html
```